extension/src/experiments/model/util.ts

```diff
@@ -1,24 +1,16 @@
 import get from 'lodash.get'
-import { formatDate } from '../../util/date'
 import { formatNumber } from '../../util/number'
 import { truncateFromLeft } from '../../util/string'
 import { splitColumnPath } from '../columns/paths'
 import { Experiment } from '../webview/contract'
 
 type Value = undefined | null | [] | string | number
 
-const isDate = (value: Value): boolean =>
```

Comment:
We had some utils that I believe were just used to handle dates in table tooltips which can now be deleted.

Reply:
Good point! I'll add the date util back and make sure one of our tests had a date value!